assets/javascripts/discourse/docker-manager-route-map.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  resource: "admin",
+
+  map() {
+    this.route(
```

Comment:
Once merged, we should remove `/admin/upgrade` from the [server_side_only list](https://github.com/discourse/discourse/blob/8efb787d8758cbe2ddc6a42b79cbdfa0e0b1909a/app/assets/javascripts/discourse/app/lib/url.js#L30-L30) in core